gamedb.yaml

```diff
@@ -5493,3 +5493,745 @@
+  id: 1295420179
+  banner: steam:912570
+  status: playable
+  notes:
+    - No controller support
+
+- name: "Helium Rain"
+  platform: gog
+  id: 1807214820
+  banner: steam:681330
+  status: playable
+  notes:
+    - Tutorial shows keyboard icons
+
+- name: "Hero of the Kingdom II"
+  platform: gog
+  id: 1666688320
+  banner: steam:346560
+  status: playable
+  notes:
+    - No controller support
+
+- name: "Ozymandias: Bronze Age Empire Sim"
+  platform: gog
+  id: 1166262945
+  banner: steam:1768280
+  status: playable
+  notes:
+    - No controller support
+
+- name: "Jazz Jackrabbit 2: The Christmas Chronicles"
+  platform: gog
+  id: 1597842603
+  banner: https://cdn2.steamgriddb.com/grid/5280fc5bd764f3387f774f273c4b3f01.png
+  poster: https://cdn2.steamgriddb.com/grid/c81632061ae1a7602021f5669357cc34.png
+  background: https://cdn2.steamgriddb.com/hero/89eec4cf449e1db057297d6f9304a99e.png
+  logo: https://cdn2.steamgriddb.com/logo/2d5c1f9d0ff0d0bc3f0465f0ff1ff653.png
+  icon: https://cdn2.steamgriddb.com/icon/d869c99656ec60fc9de27338a87b2506.ico
+  status: playable
+  notes:
+    - Limited controller support
+
+- name: "Jazz Jackrabbit 2: The Secret Files"
+  platform: gog
+  id: 1351891846
+  banner: https://cdn2.steamgriddb.com/grid/5391707bb1c266d61d0c6d1e7948faaa.png
+  poster: https://cdn2.steamgriddb.com/grid/d6f993113df8012c1868e8612a279693.png
+  background: https://cdn2.steamgriddb.com/hero/8cff9bf6694dccfc3b6a613d05d51d16.png
+  logo: https://cdn2.steamgriddb.com/logo/54cf07b091f36d3e0a2d1e5bcf65d66a.png
+  icon: https://cdn2.steamgriddb.com/icon/d6d83fd1acd1918a677906afc9bdc1d7.ico
+  status: verified
+  notes:
+    - Limited controller support
+
+- name: "The Beast Inside"
+  platform: gog
+  id: 1341028548
+  banner: steam:792300
+  status: playable
+  notes:
+    - No controller support
+
+- name: "Fallout"
+  platform: gog
+  id: 1440148836
+  banner: steam:38400
+  status: playable
+  notes:
+    - No controller support
+
+- name: "Fallout 2"
+  platform: gog
+  id: 1440151285
+  banner: steam:38410
+  status: playable
+  notes:
+    - No controller support
+
+- name: "Fallout Tactics"
+  platform: gog
+  id: 1440152063
+  banner: steam:38420
+  status: playable
+  notes:
+    - No controller support
+    - Cut Scenes are stuttering
+
+- name: "Legend of Keepers: Career of a Dungeon Manager"
+  platform: gog
+  id: 1208434383
+  banner: steam:978520
+  status: playable
+  notes:
+    - No controller support
+
+- name: "Powers in the Basement"
+  platform: gog
+  id: 1985215349
+  banner: steam:2520280
+  status: playable
+  notes:
+    - No controller support
+
+- name: "Prison Architect"
+  platform: gog
+  id: 1441974651
+  banner: steam:233450
+  status: playable
+  notes:
+    - No controller support
+
+- name: "Shadow Tactics: Blades of the Shogun"
+  platform: gog
+  id: 1601442230
+  banner: steam:418240
+  compat_tool: GE-Proton9-2
+  status: playable
+  notes:
+    - Enabling  controller with mouse
```

Comment:
What does this mean exactly? Also, there is an extraneous space.

Reply:
Game has not controller enabled on first start. Can be enabled in options.
I updated the wording herre.